engine/src/test/java/com/datatorrent/stram/cli/ApexCliTest.java

```diff
@@ -258,4 +258,26 @@ public void testMergeAppFromConfigAndAppPackage() throws Exception
 
     Assert.fail("Cli failed throw multiple apps exception.");
   }
+
+  @Test
+  public void testGetOperatorClassesCommandLineInfo() throws Exception
+  {
+    // The second and third arg values in the method call can be any strings.
+    // The unit test only checks that the corresponded values of the output parser options
+    // are equal to the same values that were passed in the method call.
+
+    String arg2 = "com.example.mydtapp.Application";
```

Comment:
Please change arg2 = "arg2" and arg3 = "arg3" (or "test1" and "test2") to avoid confusion on argument meanings. In this case, comments are not required.